packages/nx-plugin/src/generators/configure-storybook/generator.ts

```diff
@@ -16,9 +16,8 @@ export default async function (
 
   const { root: projectRoot } = project;
   await configurationGenerator(tree, {
-    name,
+    project: name,
     configureCypress: false,
-    uiFramework: '@storybook/react-webpack5',
```

Comment:
wondering why is this remove ? based on API it's required https://nx.dev/nx-api/storybook/generators/configuration#uiframework